networkx/drawing/nx_pylab.py

```diff
@@ -678,6 +681,13 @@ def draw_networkx_edges(
     # undirected graphs (for performance reasons) and use FancyArrowPatches
     # for directed graphs.
     # The `arrows` keyword can be used to override the default behavior
+
+    if not G.is_directed() and arrowstyle == None:
+        arrowstyle = "-"
+
+    if G.is_directed() and arrowstyle == None:
+        arrowstyle = "-|>"
```

Comment:
```suggestion
    if arrowstyle == None:
        if G.is_directed():
            arrowstyle = "-|>"
        else:
            arrowstyle = "-"
```

Reply:
Thank you @jarrodmillman , I committed the suggestion.